i18n/locales/source/blockly-mooc/common.json

```diff
@@ -11,7 +11,7 @@
   "abusiveUrl": "URL of the content being reported",
   "achievements": "Achievements",
   "activity": "Activity",
-  "activityHeader": "{activityName} ({activityTime} minutes)",
+  "activityHeader": "{activityName} ({activityDuration} minutes)",
```

Comment:
see https://github.com/code-dot-org/code-dot-org/pull/36914#discussion_r495268742 for why we're okay with this change